sufia-models/app/jobs/batch_update_job.rb

```diff
@@ -41,22 +42,17 @@ def update_file(gf, user)
       denied << gf
       return
     end
+    # update the file using the actor after setting the title
     gf.title = title[gf.id] if title[gf.id]
-    gf.attributes = file_attributes
-    gf.visibility= visibility
+    Sufia::GenericFile::Actor.new(gf, user).update_metadata(file_attributes, visibility)
+
+    # update the work to the same metadata as the file.
+    # NOTE: For the moment we are assuming copied metadata.  This is likely to change.
+    unless gf.generic_work.blank?
```

Comment:
@awead  In our old upload scenario there is a possibility the work gets in without a work.

Aka I upload a bunch of files, the original way.  No work is specified for them to go in. I should get a work, but I am just being careful.  You can create a file from the rails console without specifying a work at this point too.